app/assets/stylesheets/user.css.scss

```diff
@@ -26,3 +27,9 @@
     font-size: 24px;
 }
 
+#user-card {
+  border: solid 1px #363636;
+  border-radius: 5px 5px 5px 5px;
```

Comment:
Shorthand form for property `border-radius` should be written more concisely as `5px` instead of `5px 5px 5px 5px`